src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,5 +60,15 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
-
+    
```

Comment:
`isSimilar` method should be done after you designed your test. You create this method to pass the test case.